ui/src/plugins/com.google.PerfettoMcp/index.ts

```diff
@@ -0,0 +1,193 @@
+// Copyright (C) 2025 The Android Open Source Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Trace} from '../../public/trace';
+import {App} from '../../public/app';
+import {MetricVisualisation} from '../../public/plugin';
+import {PerfettoPlugin} from '../../public/plugin';
+import {McpServer} from '@modelcontextprotocol/sdk/server/mcp.js';
```

Comment:
These seem to be an issue, not sure how to use a large dependency like this in a plugin.

Reply:
What seems to be the issue?

Reply:
```
CheckCodeFormatted took >3.00s (3.93s)
Import not allowed /plugins/com.google.PerfettoMcp/index -> @modelcontextprotocol/sdk/client/index.js 
Import not allowed /plugins/com.google.PerfettoMcp/index -> @modelcontextprotocol/sdk/inMemory.js 
Import not allowed /plugins/com.google.PerfettoMcp/index -> @modelcontextprotocol/sdk/server/mcp.js 

--- Presubmit Errors Found ---
- Import violations detected in ui/ sources. See tools/check_imports
```

Reply:
@stevegolton any advice on this?